Neos.Neos/Configuration/Settings.yaml

```diff
@@ -230,12 +230,6 @@ Neos:
             editor: Neos.Neos/Inspector/Editors/DateTimeEditor
             editorOptions:
               format: d-m-Y
-          reference:
-            typeConverter: Neos\Neos\Service\Mapping\NodeReferenceConverter
-            editor: Neos.Neos/Inspector/Editors/ReferenceEditor
-          references:
-            typeConverter: Neos\Neos\Service\Mapping\NodeReferenceConverter
-            editor: Neos.Neos/Inspector/Editors/ReferencesEditor
```

Comment:
How are references converted nowadays?

Reply:
I currently have this logic in place

```
$editor = $maxAllowedItems === 1 ? 'Neos.Neos/Inspector/Editors/ReferenceEditor' : 'Neos.Neos/Inspector/Editors/ReferencesEditor';
```

but i sure could use here the configured `dataTypes.reference` or (s) instead.
About the `typeConverter`, the ui handles references totally different in the node property conversion service see: `\Neos\Neos\Ui\Domain\Service\NodePropertyConverterService::getPropertiesArray` where `\Neos\Neos\Ui\Domain\Service\NodePropertyConverterService::convertValue` is only used for properties in Neos 9 and thus the `typeConverter` has no effect for the references.

Reply:
Ah, nice. I was just curious